Codeinwp/neve
# Fix widget inputs

### Summary
- Applied the customizer field settings on widget select field

### Will affect visual aspect of the product
NO

### Test instructions
- Add archive and categories widget in any sidebar
- Check the box that says  "Display as dropdown"
- Go to Customizer -> Form Fields and change the controls there ( padding, border, border-radius, colors, etc. )
- The settings should apply on both fields and should be visible in both customizer and after you save, on front-end.


Closes #2581.
